engines/tetraedge/te/te_font3.cpp

```diff
@@ -46,7 +46,7 @@ Graphics::Font *TeFont3::getAtSize(uint size) {
 		error("TeFont3::: Couldn't open font file %s.", getAccessName().c_str());
 
 	_fontFile.seek(0);
-	Graphics::Font *newFont = Graphics::loadTTFFont(_fontFile, size, Graphics::kTTFSizeModeCharacter, 0, Graphics::kTTFRenderModeNormal);
+	Graphics::Font *newFont = Graphics::loadTTFFont(&_fontFile, DisposeAfterUse::NO, size, Graphics::kTTFSizeModeCharacter, 0, Graphics::kTTFRenderModeNormal);
```

Comment:
I'm not sure if this is correct - `_fontFile` needs to be kept open until the font is deleted.

Reply:
I think this will be ok for tetraedge, `TeFont3::unload()` clears the references to the font objects before closing `_fontFile`, so it should work.